source/installation/index.rst

```diff
@@ -0,0 +1,23 @@
+.. Copyright (C) 2020 GovReady PBC
```

Comment:
Should this be `Copyright (C) 2021 GovReady PBC` or `Copyright (C) 2020-2021 GovReady PBC`  b/c it is a new file?